.github/workflows/set-tag.yml

```diff
@@ -0,0 +1,29 @@
+name: Set npm Registry Latest Tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'Name of package'
+        default: 'holocron'
+        required: true
+        type: string
+        options:
+        - holocron
+        - holocron-module-register-webpack-plugin
+        - holocron-module-route
+        - iguazu-holocron
```

Comment:
type is string but we provide options?
`type: choice`

Reply:
is the value not a string ?

Reply:
the types can only be one of boolean, number, or string.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype